test/files/neg/t12565.scala

```diff
@@ -0,0 +1,9 @@
+// scalac: -Werror --release 8
+// javaVersion: 9+
```

Comment:
@lrytz on scala 3 I contributed `test: -jdk 9+` syntax. IIRC. Not sure if you renewed the patent application.